packages/teams-js/test/public/tasks.spec.ts

```diff
@@ -1,14 +1,11 @@
+/* eslint-disable strict-null-checks/all */
 import { app } from '../../src/public/app';
 import { TaskModuleDimension } from '../../src/public/constants';
 import { FrameContexts } from '../../src/public/constants';
 import { TaskInfo } from '../../src/public/interfaces';
 import { tasks } from '../../src/public/tasks';
 import { Utils } from '../utils';
 
-/* eslint-disable */
```

Comment:
Why was this removed and replaced with the eslint-disable on line 1?

Reply:
Same as above